package.json

```diff
@@ -87,6 +88,9 @@
     "perfect-scrollbar": "^1.3.0",
     "po-loader": "^0.4.1",
     "po2json": "^0.4.5",
+    "postcss-import": "^12.0.0",
+    "postcss-loader": "^2.1.6",
+    "postcss-preset-env": "^5.3.0",
```

Comment:
these are for Autoprefixer library

Reply:
if you use autoprefixer, you shouldn't need most of the `@mixin` that just prefix `mox-` or `webkit-`

Reply:
@brucebinary I should remove `@mixin` or just remove `webkit-` props on the mixin? If I am going to remove `mixin` then I have to modify a lot of CSS file.

Reply:
@shayanbinary sure go for it. That's why we want to introduce autoprefixer in the first place.